ERROR - 2025-09-27 23:20:39 --> mysqli_sql_exception: Table 'warehouse.users' doesn't exist in C:\xampp\htdocs\warehouse\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\warehouse\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\warehouse\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\warehouse\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\warehouse\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\warehouse\app\Database\Seeds\UserSeeder.php(16): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\warehouse\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#6 C:\xampp\htdocs\warehouse\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#7 C:\xampp\htdocs\warehouse\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#8 C:\xampp\htdocs\warehouse\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#9 C:\xampp\htdocs\warehouse\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\warehouse\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\warehouse\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
